vspace/src/main/webapp/WEB-INF/views/staff/modules/module.html

```diff
@@ -339,6 +339,19 @@ <h4>All Slides</h4>
                 </div>
             </div>              
         </div>
+        <div class="tab sequences" style="margin-top: 10px;">
+            <div style="display: flex; justify-content: space-between">
+               <h4>List of Spaces Linked to a Module</h4>
```

Comment:
Should be "Spaces linking to this Module"

Reply:
fixed